develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Transfer Assets with XCM
+description: This tutorial will guide you through the process of transferring assets between parachains using Cross-Chain Message (XCM) instructions.
+---
+
+# Transfer Assets with XCM
+
+## Introduction
+
+In the tutorial on [Open Message Passing Channels](TODO:update-path){target=\_blank} you learned how to establish a two-way communication channel between chains by sending messages to the relay chain. You can apply a similar approach to send messages that enable a local chain to manage an account on a remote chain. This tutorial will teach you how to transfer assets between chains using Cross-Chain Message (XCM) instructions.
```

Comment:
You can add the link to `Open Message Passing Channels` because its already on master